networkx/algorithms/connectivity/tests/test_molecular_connectivity.py

```diff
@@ -0,0 +1,56 @@
+"""
+    Unit tests for randic index.
+"""
+
+import networkx as nx
+import unittest
```

Comment:
Networkx uses pytest instead of unittest.

It doesn't have to be a blocker as we can rewrite the tests, but just something that should be reworked at some point.